### PR TITLE
chore(ProfitClient): Misc. cleanup before update

### PR DESCRIPTION
**Pull request by pxrl**
This is a single commit containing a collection of minor changes that were accrued whilst updating the ProfitClient profitability logic. They are separated out here to avoid cluttering the main PR.

 - Use constants for minRelayerFeePct.
 - Reorg. token data in ProfitClient test.

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/clients/ProfitClient.ts [hunk 1]
```diff
@@ -13,7 +14,9 @@ type CoinGeckoPrice = {
 };
 
 // We use wrapped ERC-20 versions instead of the native tokens such as ETH, MATIC for ease of computing prices.
+// @todo: These don't belong in the ProfitClient; they should be relocated.
 export const MATIC = "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0";
+export const USDC = "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48";
```
Comment (nicholaspai) [inline src/clients/ProfitClient.ts:19]: Should this be the checksummed address?

**Review by nicholaspai — APPROVED** [review]
One comment then LGTM

File: src/clients/ProfitClient.ts (hunk 1 above)
Comment (mrice32) [reply src/clients/ProfitClient.ts:19]: Great catch!!

**Pull request merged**